From 43f49349064cf3e410a4cced9dcd36cec806e188 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Thu, 11 Jan 2024 09:25:48 +0100
Subject: [PATCH 0471/1695] core: scmi-server: set CFG_SCPFW_MOD_* CMake' build
 directive

Set the OP-TEE boolean config switches related to SCP-firmware modules
as CMake directive for SCP-firmware configuration. The change allows
to easy how module can be conditionally embedded in SCP-firmware
library.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git e762965ca8b141db8524d7b5b93e7bc33a1022eb core: scmi-server: set CFG_SCPFW_MOD_* CMake' build directive]
Change-Id: Iad4c56df7f20c6b565bc5acaaa0f180106c0afff
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/lib/scmi-server/sub.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/lib/scmi-server/sub.mk b/core/lib/scmi-server/sub.mk
index 15834c14b..07467c02a 100644
--- a/core/lib/scmi-server/sub.mk
+++ b/core/lib/scmi-server/sub.mk
@@ -175,6 +175,7 @@ endif
 endif
 
 cflags-lib-$(CFG_SCPFW_MOD_$4) += -DBUILD_HAS_MOD_$4
+scpfw-cmake-flags-y += -DCFG_SCPFW_MOD_$4=$(CFG_SCPFW_MOD_$4)
 endef
 
 define scpfw-embed-generic-module
-- 
2.47.2

